runtime/src/rad_claims.rs

```diff
@@ -215,8 +218,9 @@ impl<T: Trait> frame_support::unsigned::ValidateUnsigned for Module<T> {
         if let Call::claim(account_id, amount, sorted_hashes) = call {
             // Check that proofs are valid with a root that exists in the root hash storage
             if Self::verify_proofs(account_id, amount, sorted_hashes.into()) {
-                return ValidTransaction::with_tag_prefix("RADclaim")
+                return ValidTransaction::with_tag_prefix("RadClaims")
                     .priority(T::UnsignedPriority::get())
+                    .and_provides((account_id, amount, sorted_hashes))
```

Comment:
If we do not provide these tags, then other nodes will not be able to identify the correct tx to propagate.